Rakefile

```diff
@@ -5,8 +5,6 @@ require 'rubocop/rake_task'
 require 'jekyll'
 require 'jsonlint/rake_task'
 
-task default: %w[proof verify jsonlint rubocop]
-
```

Comment:
This breaks "bundle exec rake" (as given in `CONTRIBUTING.md`) since there are no default tasks:
>rake aborted!
>Don't know how to build task 'default' (see --tasks)